design/release-notes/release-0.11/draft-release-notes.md

```diff
@@ -60,8 +60,22 @@ A table of resources and their old and new `apiVersion`s:
 You must also make sure to update all references to cert-manager in annotations to their
 new prefix:
 
-| Annotation | Affected resources | New annotation |
-|------------|--------------------|----------------|
+| Annotation                                   | Affected resources | New annotation                              |
+|----------------------------------------------|--------------------|---------------------------------------------|
+| certmanager.k8s.io/acme-http01-edit-in-place | Ingress            | acme.cert-manager.io/http01-edit-in-place   |
+| certmanager.k8s.io/acme-http01-ingress-class | Ingress            | acme.cert-manager.io/http01-ingress-class   |
+| certmanager.k8s.io/issuer                    | Ingress            | cert-manager.io/issuer                      |
+| certmanager.k8s.io/cluster-issuer            | Ingress            | cert-manager.io/cluster-issuer              |
+| certmanager.k8s.io/acme-challenge-type       | Ingress            | DEPRECATED                                  |
+| certmanager.k8s.io/acme-dns01-provider       | Ingress            | DEPRECATED                                  |
+| certmanager.k8s.io/alt-names                 | Ingress            | cert-manager.io/alt-names                   |
+| certmanager.k8s.io/ip-sans                   | Ingress            | cert-manager.io/ip-sans                     |
+| certmanager.k8s.io/common-name               | Ingress            | cert-manager.io/common-name                 |
+| certmanager.k8s.io/issuer-name               | Ingress            | cert-manager.io/issuer-name                 |
+| certmanager.k8s.io/issuer-kind               | Ingress            | cert-manager.io/issuer-kind                 |
```

Comment:
`issuer-name`, `issuer-name` and `issuer-group` *are* valid on Ingress resources, but these same keys are *also* used on Secret resources to denote where the certificate came from.

(you've also not mentioned the new issuer-group annotation here)